docs/api.md

```diff
@@ -2607,6 +2698,12 @@ Contains the URL of the response.
 
 Get the browser the target belongs to.
 
+#### target.browserContext()
+
+- returns: <[BrowserContext]>
+
+Get the browser context the target belongs to.
```

Comment:
The browser context this target belongs to.

Reply:
Done.